src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditCopyUiTest.java

```diff
@@ -38,25 +39,31 @@ public void allTests() throws Exception{
         
         ______TS("Submit empty course list");
         feedbackEditPage.clickFsCopyButton();
-        feedbackEditPage.waitForModalToLoad();
+        feedbackEditPage.fsCopyToModal.waitForModalToLoad();
 
         // Full HTML verification already done in InstructorFeedbackEditPageUiTest
         feedbackEditPage.verifyHtmlMainContent("/instructorFeedbackEditCopyPage.html");
         
-        feedbackEditPage.clickFsCopySubmitButton();
-        feedbackEditPage.verifyStatus(Const.StatusMessages.FEEDBACK_SESSION_COPY_NONESELECTED);
+        feedbackEditPage.fsCopyToModal.clickSubmitButton();
+        feedbackEditPage.fsCopyToModal.waitForFormSubmissionStatusMessageVisibility();
+        feedbackEditPage.fsCopyToModal.verifyStatusContainsErrorHtmlClasses();
+        feedbackEditPage.fsCopyToModal.verifyStatusMessage(Const.StatusMessages.FEEDBACK_SESSION_COPY_NONESELECTED);
         
         
         ______TS("Copying fails due to fs with same name in course selected");
         feedbackEditPage.clickFsCopyButton();
-        feedbackEditPage.waitForModalToLoad();
-        feedbackEditPage.fillCopyToOtherCoursesForm(feedbackSessionName);
+        feedbackEditPage.fsCopyToModal.waitForModalToLoad();
+        feedbackEditPage.fsCopyToModal.fillFormWithAllCoursesSelected(feedbackSessionName);
         
-        feedbackEditPage.clickFsCopySubmitButton();
+        feedbackEditPage.fsCopyToModal.clickSubmitButton();
+        feedbackEditPage.fsCopyToModal.waitForFormSubmissionStatusMessageVisibility();
+        assertTrue(feedbackEditPage.fsCopyToModal.isFormSubmissionStatusMessageVisible());
+        feedbackEditPage.fsCopyToModal.verifyStatusContainsErrorHtmlClasses();
+        
+        feedbackEditPage.fsCopyToModal
+                        .verifyStatusMessage("A feedback session with the name \"" + feedbackSessionName + "\" "
+                                          + "already exists in the following course(s): FeedbackEditCopy.CS2104.");
```

Comment:
Why the entire string instead of the previous `FEEDBACK_SESSION_COPY_ALREADYEXISTS` constant?